src/workerd/api/sql-test.js

```diff
@@ -1,13 +1,28 @@
 import * as assert from 'node:assert'
 
-function requireException(callback, expectStr) {
+function requireException(callback, expectStr, errCheck = () => {}) {
```

Comment:
Nit: since you're importing `node:assert`... you could be using `assert.throws(...)` instead, perhaps?

Reply:
Will fix this sort of thing up in a separate PR